Quick-Start [DA]/plpgsql.md

```diff
@@ -0,0 +1,271 @@
+# Procedural languages:
+Procedural languages in PostgreSQL provide a powerful way to extend the database's functionality by writing functions and procedures in languages other than SQL. These are natively supported by PostgreSQL and can be used directly within the database environment. These languages, also known as PL languages, offer developers the flexibility to implement complex business logic and computations directly within the database.
+
+Key points about procedural languages in PostgreSQL:
+
+1. **Extended Functionality:** Procedural languages enable developers to write custom functions and procedures in languages like PL/pgSQL, PL/Python, PL/Tcl, PL/Perl, PL/R, and more. This allows for the creation of complex computations and data processing within the database.
+
+2. **Control Flow and Logic:** Procedural languages offer control flow constructs like loops, conditionals, and exception handling, making it easier to implement business logic that goes beyond the capabilities of SQL.
+
+3. **Custom Aggregates and Window Functions:** Procedural languages allow developers to define custom aggregates and window functions, enabling advanced data analysis and processing directly within the database.
```

Comment:
Also triggers! Suggest linking to the docs for these features, in case the reader wonders what a custom aggregate or window function is and wants to find out.